ace.d.ts

```diff
@@ -152,7 +152,7 @@ export namespace Ace {
   }
 
   export interface EditSessionOptions {
-    wrap: string | number;
+    wrap: "off" | "free" | "printmargin" | boolean | number;
```

Comment:
here the update @nightwing 

I know that in practice there are more values that can be used here (for example: "null", "{}", or as you said a string that represents a number). However I think these are not really.. expected usages, right? For the type definition I think we should only include the expected ones.

I put a test here so you can see for more detail what pass and not pass this type definition (I hope it helps your review :D):

```
type Wrap = "off" | "free" | "printmargin" | boolean | number;

// good values that users should use
const valids: Wrap[] = [
    "off", "free", "printmargin",
    true, false,
    -1, 80, 100, 120
];...
```

[Playground Link](https://www.typescriptlang.org/play?#code/C4TwDgpgBA6gTgQzFAvFARAewGbfVAHw2zggnyPTDgEsA7YAWwTgHN6KoAjTTAGwgI6hKHQCujLhDgBuAFByA9IqiteAEygA3BHzEQAzlGAALBMChiD0owZOYxfTVYhyAxpjoGLOvjXUGAFywiGAA2gC6qFBhclDxGDh4ADTEpOSpVLQMzGwcyXEJwHD6qdi61gUJUAC0AIypABwADKl1za1QdQBMzXIR8koqpobQzCBQAO6YcADWUPRQ1AhuwDRu0FJuCC5T0J58E24mEG7zAAY02FAAFACEvvoAlOdD3GIWIxN2Dk6imBYpAthKBIFB1BBsPQaGtPEF3HCLPRfP4giEkJFomEsCkMC10KkAN4AX1SkVSNyeqAAfFASZkCaJHHwBgplBgxHQIVC6BB1PgbjNLF4IMAqTQjGYuQJNFoaAhjCcoAAFFhrXQAHgAdDradzobC6EA)

Reply:
Thanks!